src/test/java/com/amazon/ion/bytecode/bin11/ByteArrayBytecodeGenerator11Test.kt

```diff
@@ -0,0 +1,138 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package com.amazon.ion.bytecode.bin11
+
+import com.amazon.ion.TextToBinaryUtils.hexStringToByteArray
+import com.amazon.ion.Timestamp
+import com.amazon.ion.bytecode.GeneratorTestUtil.shouldGenerate
+import com.amazon.ion.bytecode.ir.Instructions
+import com.amazon.ion.bytecode.ir.Instructions.packInstructionData
+import com.amazon.ion.impl.bin.PrimitiveEncoder
+import org.junit.jupiter.api.Assertions.assertArrayEquals
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.CsvSource
+import org.junit.jupiter.params.provider.ValueSource
+import java.nio.charset.StandardCharsets
+
+internal object ByteArrayBytecodeGenerator11Test {
+
+    @ParameterizedTest
+    @ValueSource(
+        strings = [
+            "64 4F 97 21 C5 " + // int32 -987654321
+                "86 35 7D CB 12 2E 22 1B " + // short TS reference to 2023-10-15T11:22:33.444555-00:00
+                "8F 0C " + // null struct
+                "6A " + // float 0e0
+                "6D 18 2D 44 54 FB 21 09 40 " + // float64 3.141592653589793
+                "FE 31 49 20 61 70 70 6c 61 75 64 20 79 6f 75 72 20 63 75 72 69 6f 73 69 74 79 " + // 24-byte blob
+                "6F " // false
+        ]
+    )
+    fun `generator can compile input containing multiple simple opcodes`(inputBytesString: String) {
```

Comment:
A parameterized test looks a little fishy when there's only one case provided.
Also, we have a utility function that allows you to have comments in hex strings so that we can use kotlin multiline strings instead of string concatenation. It just makes it a little easier to read.
```suggestion
    @Test
    fun `generator can compile input containing multiple simple opcodes`() {
        val inputBytesString = """
            64 4F 97 21 C5             | int -987654321
            86 35 7D CB 12 2E 22 1B    | timestamp 2023-10-15T11:22:33.444555-00:00
            8F 0C                      | null.struct
            6A                         | float 0e0
            6D 18 2D 44 54 FB 21 09 40 | float 3.141592653589793
            FE 31                      | 24-byte blob
            49 20 61 70 70 6c 61 75    |
            64 20 79 6f 75 72 20 63    |
            75 72 69 6f 73 69 74 79    |
            6F                         | false
        """.cleanCommentedHexBytes()
```

Reply:
Forgot about that helper, good call